Update customer-key-tenant-level.md

brdesai opened the pull request:
Correcting information related to DEP behavior and cmdlets.

brdesai committed:
Update customer-key-tenant-level.md

Correcting information related to DEP behavior and cmdlets.

yogkumgit commented:
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

JohanFreelancer9 reviewed (approved):
Copy review.

KCCross commented on microsoft-365/compliance/customer-key-tenant-level.md:
```diff
@@ -21,6 +21,7 @@ description: "Learn how to set up Customer Key for all data within your Microsof
 
 Using keys you provide, you can create a data encryption policy (DEP) and assign it to the tenant. The DEP encrypts data across the tenant for these workloads:
 
+- Exchange Online
```
This is confusing. The last bullet identifies the specific information that CK encrypts for EXO at the tenant level. Adding a blanket statement about EXO here would be confusing. Particularly since EXO is also covered by customer key at the application level. Be specific.

KCCross commented on microsoft-365/compliance/customer-key-tenant-level.md:
```diff
@@ -43,8 +44,7 @@ Examples:
 
-For Exchange Online, you can create a DEP that encrypts one or more user mailboxes with Customer Key. When you create a tenant-level policy, that policy will not encrypt the encrypted mailboxes. However,  the tenant-level key will encrypt the mailboxes that are not affected by a DEP already.
-
+For Exchange Online, if you have already assigned Customer Key DEPs to individual mailboxes, the tenant level policy will not override those DEPs. The tenant-level policy will only encrypt mailboxes that are not assigned a mailbox level DEP already.
```
tenant level 
change to
tenant-level

will not
change to
won't

KCCross commented on microsoft-365/compliance/customer-key-tenant-level.md:
```diff
@@ -363,13 +363,13 @@ Parameters:
 |-Identity|Specifies the data encryption policy that you want to modify.|N|
 |-Refresh|Use the Refresh switch to update the data encryption policy after you rotate any of the associated keys in the Azure Key Vault. You don't need to specify a value with this switch.|Y|
 |-Enabled|The Enabled parameter enables or disable the data encryption policy. Before you disable a policy, you must unassign it from your tenant. Valid values are:</br > $true: The policy is enabled</br > $false: The policy is disabled.|Y|
-|-Name|The Name parameter specifies the unique name for the data encryption policy.|Y
```
THANK YOU FOR CATCHING THIS!!!!!!

KCCross commented on microsoft-365/compliance/customer-key-tenant-level.md:
````diff
@@ -363,13 +363,13 @@ Parameters:
 
 ```powershell
-Get-M365DataAtRestEncryptionPolicy [-Identity] < M365DataAtRestEncryptionPolicy DataEncryptionPolicyIdParameter>
+Get-M365DataAtRestEncryptionPolicy [-Identity] <M365DataAtRestEncryptionPolicy DataEncryptionPolicyIdParameter>
````
ALSO THANK YOU FOR CATCHING THIS!!!!!

Commit:
Change to the coverage list

- Exchange Online 
this entry was too broad. Reduced the scope to identify just the unencrypted mailboxes.

KCCross commented on microsoft-365/compliance/customer-key-tenant-level.md:
```diff
@@ -28,6 +28,7 @@ Using keys you provide, you can create a data encryption policy (DEP) and assign
 - Teams chat suggestions by Cortana
 - Teams status messages
 - User and signal information for Exchange Online
+- Exchange Online mailboxes that are not already encrypted Customer Key DEPs at the service level.
```
Brijesh, please review the change I made here.
I removed 
- Exchange Online
and added this line.

Commit:
Editorial updates

levi8282 commented:
Please unsubscribe me from your mailing list

On Wed, Mar 10, 2021 at 5:59 PM KC Cross ***@***.***> wrote:

> Merged #4274
> <https://github.com/MicrosoftDocs/microsoft-365-docs/pull/4274> into
> public.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/MicrosoftDocs/microsoft-365-docs/pull/4274#event-4439797418>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOGLPBYFDHWQ3NRGYKK6BTDTC7TUVANCNFSM4YKNWVZQ>
> .
>
